package/package.json

```diff
@@ -64,7 +64,7 @@
 	"devDependencies": {
 		"@types/node": "^20.11.0",
 		"typescript": "^5.3.3",
-		"vite": "^5.0.11",
+		"vite": "^5.1.0",
```

Comment:
Needed to avoid weird typing conflicts

Reply:
I pulled it and try and it looks good.

Reply:
Nice, would be great if we could declare that something is not necessarily a peer, but if it is it should match the version.

Reply:
What do you mean? It's not a peer but a dev dep rn but I'm probably missing something

Reply:
If it were a peer it would not have the version conflict issue, but then it would require projects using AIK to install vite directly as well.

It would be nice if there was a way to just declare that a dependency should be deduplicated with a peer if present, but installed as transitive if not. There isn't, sadly.